src/main/webapp/WEB-INF/jsp/staleUserResetPassword.jsp

```diff
@@ -12,9 +12,13 @@
              <p>
                  <spring:message code="public.reset.password.stale.users.warning.fix" />
```

Comment:
This message is still confusing I think
"Please fix the following"
The user has nothing to fix on the page.
Should we remove it?

Reply:
I'm also wondering whether this needs to be in an error summary box at all?

Reply:
it can just be the message I guess, maybe UX team can weigh in

Reply:
design updated